openreview/venue_request/venue_request.py

```diff
@@ -498,13 +498,11 @@ def setup_request_form(self):
                 'order': 16
             },
             'Paper Matching': {
-                'description': 'Choose options for assigning papers to reviewers. If using the OpenReview Paper Matching System, see the top of the page for a description of each feature type.',
+                'description': 'Choose options for assigning papers to reviewers (and ACs, if present). If using the OpenReview Paper Matching System, see the top of the page for a description of each feature type.',
                 'values-checkbox': [
-                    'Organizers will assign papers manually',
                     'Reviewer Bid Scores',
                     'Reviewer Recommendation Scores',
-                    'OpenReview Affinity',
-                    'TPMS'
+                    'OpenReview Affinity'
                 ],
                 'order': 17,
                 'required': True,
```

Comment:
If manual assignments implies no selection, then required must be False. @celestemartinez can you check this?

Reply:
You are right, I also added instruction to deselect all options if manually assigning reviewers.